src/content/blog/windows-ddev-setup.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Setting up a Windows Machine for DDEV"
+pubDate: 2024-11-04
+#modifiedDate: 2024-10-17
+summary: Setting up a new Windows machine for DDEV maintenance or development is pretty easy. Here are my opinionated steps.
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/11/windows-install-blog-logos.png
+  alt: Windows, DDEV, Ubuntu logos demonstrating setting up a Windows machine for DDEV.
+categories:
+  - DevOps
+---
+
+I've recently set up a few Windows machines for DDEV maintenance and development, and wanted to share how I do it. It's surprisingly easy. My approach here is opinionated, but it works for me. You'll do things a little differently I'm sure.
+
+Two recent Windows machines I set up were the new ARM64/Qualcomm/CoPilot variety. They were excellent and fast and had great battery life. There's very little I had to do differently with them, but I'll mention in the steps when there was something different. (I was surprised by the lack of a fingerprint sensor on both, but the Windows Hello facial recognition was quite fast. There is no ARM64 Discord app, and the AMD64 one had horrific performance.)
+
+1. Remove unwanted applications. I usually start by removing a bunch of bloatware. 
```

Comment:
it might be helpful to add a few examples here? it might be even insightful if you consider some apps as bloatware that a user wasnt even aware of? an opportunity to educate?

Reply:
```suggestion
1. Remove unwanted applications like Solitaire and MS Office. I usually start by removing a bunch of bloatware. 
```